src/modules/user-settings/components/editor-picker.tsx

```diff
@@ -23,16 +24,43 @@ export const EditorPicker = ( { value, onChange, disabled }: EditorPickerProps )
 		} ),
 	} );
 
+	const hasEditorInstalled = installedEditors.length > 0;
+
+	const renderEditorRecommendation = () => {
+		if ( hasEditorInstalled ) {
+			return null;
+		}
+
+		return (
+			<p className="text-gray-500">
+				{ /* translators: "Visual Studio Code" is a trademarked brand name by Microsoft. Do not translate brand names. */ }
+				{ createInterpolateElement( __( 'We recommend using <a>Visual Studio Code ↗ </a>.' ), {
+					a: (
+						<Button
+							onClick={ () => getIpcApi().openURL( 'https://code.visualstudio.com/' ) }
+							variant="link"
+						/>
+					),
+				} ) }
+			</p>
+		);
+	};
+
 	return (
 		<SettingsFormField label={ __( 'Code editor' ) }>
 			<SelectControl
 				value={ value }
 				onChange={ ( newValue ) => onChange( newValue as SupportedEditor ) }
 				__nextHasNoMarginBottom
 				__next40pxDefaultSize
-				disabled={ disabled }
+				disabled={ ! value && ! hasEditorInstalled }
```

Comment:
Disabling the control altogether when no supported editors are installed doesn't strike me as an obviously better user experience. The dropdown menu already has the `Not installed` optgroup, and it might be interesting to know which editors Studio supports. 

Granted, it's now clear that VS Code is supported, but I still don't really see it as an improvement to disable the control in this way.